cmake/rs_addons_config.cmake

```diff
@@ -0,0 +1,6 @@
+set(rs_addons_NAMESPACE           rs_addons)
+set(rs_addons_TYPESYSTEM_CPP_PATH /home/rakib/catkin_ws/src/rs_addons/include/rs_addons/types)
+set(rs_addons_TYPESYSTEM_XML_PATH /home/rakib/catkin_ws/src/rs_addons/descriptors/typesystem)
+set(rs_addons_ANNOTATOR_PATH      /home/rakib/catkin_ws/src/rs_addons/descriptors/annotators)
+set(rs_addons_ENGINE_PATH         /home/rakib/catkin_ws/src/rs_addons/descriptors/analysis_engines)
+
```

Comment:
this file should not be added to the repo. It gets auto generated during compilations